lib/ansible/plugins/connection/network_cli.py

```diff
@@ -352,7 +352,11 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
     @property
     def paramiko_conn(self):
         if self._paramiko_conn is None:
-            self._paramiko_conn = connection_loader.get('paramiko', self._play_context, '/dev/null')
+            for fq_conn_name in ['ansible.netcommon.paramiko', 'paramiko']:
+                self._paramiko_conn = connection_loader.get(fq_conn_name, self._play_context, '/dev/null')
+                if self._paramiko_conn is not None:
+                    break
```

Comment:
This should probably have something along the lines of

```suggestion
                    break
            else:
                raise AnsibleError(missing_required_lib("collection ansible.netcommon"))
```

Reply:
imo this is not required as we are certain that parmiko_ssh will be either in one of the locations for sure.

Reply:
...right, we're already in netcommon. Carry on, then.